web/src/Logout.vue

```diff
@@ -6,8 +6,8 @@ import {useRouter} from 'vue-router'
 
 export default {
   mounted() {
-    this.$cookies.remove("session")
```

Comment:
if cookies aren't used any more, `vue3-cookies` package can be removed (and the plugin in Vue app init as well)